medusa/providers/torrent/html/limetorrents.py

```diff
@@ -96,7 +97,8 @@ def search(self, search_strings, age=0, ep_obj=None):
 
                     search_url = self.urls['search'].format(query=search_string)
                 else:
-                    search_url = self.urls['rss'].format(page=1)
+                    search_url = self.urls['tv']
+                    #search_url = self.urls['rss'].format(page=1)
```

Comment:
I have hopes this is just something temporary, so i'm leaving it here to change back.